outlines/welcome.md

```diff
@@ -0,0 +1,13 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework
```

Comment:
It might be too ambitious, but we can say parts of it are also a template for how we think app development in general should work, and could be transferable to other frameworks. This could be good for posting to Hacker News lol